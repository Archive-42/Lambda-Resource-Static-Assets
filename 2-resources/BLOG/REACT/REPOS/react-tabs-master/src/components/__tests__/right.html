<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>__tests__</title>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/bgoonz/GIT-CDN-FILES/right.css">
</head>

<body>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><td colspan="3" class="header-row"><b>Folders</b></td></tr>
<tr><td style="text-align: center"><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/../right.html" target="rframe">&lt;parent&gt;</a></td><td style="text-align: center">&nbsp;<a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/right.html" target="rframe">__snapshots__</a></td><td style="text-align: center">&nbsp;<a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/helpers/right.html" target="rframe">helpers</a></td></tr>
</table>
<br>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/right.html" target="_parent">right.html</a></td><td align="right">2,890</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/TabPanel-test.js" target="_parent">TabPanel-test.js</a></td><td align="right">1,794</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/Tabs-test.js" target="_parent">Tabs-test.js</a></td><td align="right">13,890</td></tr>
<tr class="row2"><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/TabList-test.js" target="_parent">TabList-test.js</a></td><td align="right">2,764</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/Tabs-node-test.js" target="_parent">Tabs-nod...st.js</a></td><td align="right">1,048</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/Tab-test.js" target="_parent">Tab-test.js</a></td><td align="right">1,807</td></tr>
</table>

</body>
</html>