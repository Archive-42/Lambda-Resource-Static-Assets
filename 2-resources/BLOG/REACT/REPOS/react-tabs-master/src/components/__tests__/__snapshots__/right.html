<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>__snapshots__</title>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/bgoonz/GIT-CDN-FILES/right.css">
</head>

<body>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><td colspan="3" class="header-row"><b>Folders</b></td></tr>
<tr><td style="text-align: center"><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/../right.html" target="rframe">&lt;parent&gt;</a></td></tr>
</table>
<br>

<table border="0" cellspacing="1" cellpadding="2" summary="file table" class="file-table centered">
<tr><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th><th><b>File</b></th><th><b>Size</b></th></tr>
<tr><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/right.html" >right.html</a></td><td align="right">2,314</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/TabPanel-test.js.snap" >TabPanel....snap</a></td><td align="right">1,519</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/Tab-test.js.snap" >Tab-test.js.snap</a></td><td align="right">2,471</td></tr>
<tr class="row2"><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/TabList-test.js.snap" >TabList-....snap</a></td><td align="right">5,001</td><td><a href="File://C:/Users/bryan/Downloads/STATIC-ASSETS/2-resources/BLOG/REACT/REPOS/react-tabs-master/src/components/__tests__/__snapshots__/Tabs-test.js.snap" >Tabs-tes....snap</a></td><td align="right">21,661</td><td></td><td></td></tr>
</table>

</body>
</html>